bpabi32/bpabi32.rst

```diff
@@ -0,0 +1,2967 @@
+   |                                 | PC-relative [DSO] or absolute, during linking. | by a dynamic relocation [bpabi32-note1_].                     |
+   | RO → RO                         |                                                |                                                               |
+   +---------------------------------+------------------------------------------------+---------------------------------------------------------------+
+   | References                      | References from RO → RW/ZI are resolved,       | Each not SB-relative reference RO → RW/ZI must be relocated   |
+   |                                 | PC-relative [DSO] or absolute, during linking. | by a dynamic relocation [bpabi32-note2_].                     |
+   | RO → RW/ZI                      | (Dynamically relocated GOT entries remain).    |                                                               |
+   +---------------------------------+------------------------------------------------+---------------------------------------------------------------+
+   | References                      | Each reference RW → RO must be                 | Each reference RW → RO must be relocated w.r.t the base of    |
+   |                                 | relocated w.r.t the base of the RO segment.    | the RO segment [bpabi32-note1_].                              |
+   | RW → RO                         |                                                |                                                               |
+   +---------------------------------+------------------------------------------------+---------------------------------------------------------------+
+   | References                      | Each reference RW → RW/ZI must be relocated    | Each reference RW → RW/ZI must be relocated w.r.t the RW      |
+   |                                 | w.r.t the base of the RO segment (there is     | segment base [bpabi32-note3_].                                |
+   | RW → RW/ZI                      | only one base address).                        |                                                               |
+   +---------------------------------+------------------------------------------------+---------------------------------------------------------------+
+   | References to ZI                | References to BSS are fully resolved,          | Each reference to ZI must by relocated by a ZI-relative       |
+   |                                 | PC-relative DSO] or absolute, by the static    | dynamic relocation [bpabi32-note4_].                          |
+   |                                 | linker.                                        |                                                               |
+   +---------------------------------+------------------------------------------------+---------------------------------------------------------------+
+
+.. note::
```

Comment:
In the PDF, the content of the note crosses the outline of the note.

Reply:
I see that, but what can we do about it? I don't know of any RST feature that asks that not to happen. (Surely it wasn't asked for on purpose in the first place.)

Reply:
the hacky way around this is to put a page-break before the note